packages/react-reconciler/src/__tests__/ReactUpdaters-test.internal.js

```diff
@@ -266,9 +274,6 @@ describe('updaters', () => {
     await waitForAll([]);
   });
 
-  // This test should be convertable to createRoot but the allScheduledTypes assertions are no longer the same
-  // So I'm leaving it in legacy mode for now and just disabling if legacy mode is turned off
-  // @gate !disableLegacyMode
   it('should cover suspense pings', async () => {
```

Comment:
The original issue in the test was that the updater was on the Default lane, the updater  got pinged, and then we said we should render on a Retry lane (`markRootPinged`). This left the updater dangling in pending updaters since we only move the pending updaters to memoized on the lane we actually render (Retry lane).